inabox/deploy/cmd/main.go

```diff
@@ -85,6 +85,15 @@ func DeployAll(ctx *cli.Context) error {
 		return fmt.Errorf("deploy experiment: %w", err)
 	}
 
+	// Enable interval mining after contract deployment
+	// This gives us instant mining during deployment (for speed),
+	// but regular block intervals after deployment (for testing)
```

Comment:
anvil has a hybrid mining mode which we could just start anvil with and call it a day afaiu.